src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(EnumFigure.values().length);
```

Comment:
[Think about which variables should be local in the method and which should be class-level](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)